tests/integration/components/lt-body-test.js

```diff
@@ -33,7 +33,7 @@ test('row selection - enable or disable', function(assert) {
 
   this.render(hbs `{{lt-body table=table sharedOptions=sharedOptions canSelect=canSelect}}`);
 
-  let row = this.$('tr:first');
+  let row = this.$('tr:not(.lt-scaffolding-row):first');
```

Comment:
That is... correct. 😨 

I could swear, that I've written these. 🤔  
Gonna push the tomorrow from the office. ☺️